docs/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.md

```diff
@@ -6,9 +6,178 @@ jupyter:
+
+Also notice that only a single CTE is created from the `electric` table and converted into a `pandas` DataFrame for the `scatterplot()` function. Like before, the `interact` function is called to visualize the scatterplot with the `Dropdown` widget.
+
+Read the insights gained from this plot [below](https://ploomber-sql.readthedocs.io/en/latest/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.html#id3)
+
+#### `ggplot` API Histogram
+
+The boxplot of $CO_2$ emissions of all types of vehicles is a good starting point for exploring the distribution of $CO_2$ emissions. However, it is also important to visualize the distribution of $CO_2$ emissions by vehicle type (fuel-only, electric, and hybrid) and fuel type. Therefore, the fourth visualization is a `ggplot` histogram of $CO_2$ emissions of fuel-only cars, electric cars, and hybrid cars. This histogram is similar to the [example](https://ploomber-sql.readthedocs.io/en/latest/visualizing-your-sql-queries/plotting-with-ggplot.html#categorical-histogram-with-select-widget) from the Visualizing Your SQL Queries module. Three widgets, two Selection and an `IntSlider`, are employed for this visualization. The `RadioButtons` widget is used for mapping `fill` with either vehicle type or fuel type. The `Dropdown` widget allows the user to choose any of the five `cmap` options, which changes the color of the bars. The `IntSlider` allows selecting the number of bins, between 1 and 20, for the histogram. Unlike the boxplot, this visualization makes use of one CTE, created from the `all_vehicles` table.
+
+Read the insights gained from this plot [below](https://ploomber-sql.readthedocs.io/en/latest/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.html#id4)
+
+#### `Seaborn` Boxplot
+
+The dataset contains five clean fuel (electric and hybrid) US car manufacturers. To assesses the cleanest, a (grouped) boxplot is visualized of $CO_2$ emissions by vehicle type and transmission type. Similar to the `scatterplot()` explained above, the `hue` attribute is integrated with a `Dropdown` widget to toggle between vehicle type, transmission type, or no `hue`. The seaborn `boxplot()` function and widget building process is also similar to the `scatterplot()` above.
+
+Read the insights gained from this plot [below](https://ploomber-sql.readthedocs.io/en/latest/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.html#id5)
+
+## Launching the Dashboard Locally
+
+Whether your Jupyter Notebook is incomplete or complete, `Voila` will still be able to render it locally on the web! This will help you in prototyping your dashboard and fit it exactly to your needs. To launch the dashboard, run the following command in your terminal, given that you are in the same directory as the `dashboard.ipynb` file:
```

Comment:
It would be good to explain how this is packaged

1. Directory structure
2. Where in the directory it is located
3. Dependencies 
4. Do they need to run the pipeline before? It still feels disjointed

Reply:
Added new sub sections for the four points listed in latest commit